mailpoet/assets/css/src/components-automation/_statistics.scss

```diff
@@ -58,6 +59,10 @@
       font-size: 12px;
       font-weight: 400;
       line-height: 16px;
+
+      &.display-after {
+        order: 1;
+      }
```

Comment:
:clap: 

this is how to solve the problem from the number on one screen is on top, at the other on bottom. Didn't think about a CSS solution when working on it. Thank you!